packages/components/addon/components/hds/code-block/description.hbs

```diff
@@ -0,0 +1,8 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Hds::Text::Body @tag="div" @size="100" class="hds-code-block__description" ...attributes>
```

Comment:
My normal preference is to use semantic elements whenever possible to contain text (so I would normally use a p tag here) but I didn't know what we would want to do. (I would use a header tag for the title also but we don't know where it could appear in the hierarchy so I could perhaps use another p tag. That was part of what I wasn't sure about and why I just used divs to start.)

Reply:
Pushing an update to change them both to use `p`.